WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1187,6 +1210,29 @@ public void onClick(DialogInterface dialog, int which) {
         builder.show();
     }
 
+    private void exportSite() {
+        final Blog currentBlog = WordPress.getCurrentBlog();
```

Comment:
I recommend adding `if (!NetworkUtils.checkConnection(getActivity())) return;` before making the network call. Even though site settings won't appear if there's no connection, there's no guarantee that won't change, and it's possible the connection could exist when settings appears but is lost by the time the user chooses to export.